crates/nargo/src/cli/verify_cmd.rs

```diff
@@ -112,3 +83,38 @@ pub fn add_dummy_setpub_arr(abi: &mut Abi) {
     // ensuring there will be no conflicts
     abi.parameters.push((RESERVED_PUBLIC_ARR.into(), dummy_arr));
 }
+
+pub fn verify_hlp(
```

Comment:
Lets rename verify_hlp to `verify_with_path`

Reply:
Done